web-components/src/[sandbox]/examples/alert-banner.ts

```diff
@@ -61,12 +68,30 @@ export class AlertBannerTemplateSandbox extends LitElement {
       <div style="margin-bottom: 15px;">
         <md-button variant="secondary" @click=${() => this.openAlert("default")}>Trigger Default Alert</md-button>
       </div>
+      <div style="margin-bottom: 15px;">
+        <md-button variant="secondary" @click=${() => this.openAlert("refresh")}
+          >Trigger Refresh Alert (inline styled icons and button)</md-button
+        >
+      </div>
       <div style="margin-bottom: 15px;">
         <md-button variant="secondary" @click=${() => this.openAlert("success")}>Trigger Success Alert</md-button>
       </div>
       <md-alert-banner ?show=${this.default} type="default" @alertBanner-hide=${() => this.hideAlert("default")}>
         Test Alert Message
       </md-alert-banner>
+      <md-alert-banner ?show=${this.refresh} type="default" @alertBanner-hide=${() => this.hideAlert("refresh")}>
+        <md-icon name="info-badge-filled" iconSet="momentumDesign" size="16"> </md-icon>
+        Refresh to see 5 new interactions
+        <md-button variant="ghostInheritTextColor" circle size="24">
+          <md-icon
+            class="md-alert-banner__text"
+            name="refresh-bold"
+            iconSet="momentumDesign"
+            size="16"
+            style="line-height: 16px"
+          ></md-icon>
```

Comment:
we should never use inline styles and don't think the line-height should be necessary